js/adapt-assessmentArticleModel.js

```diff
@@ -363,11 +372,14 @@ define([
 
       var isPercentageBased = assessmentConfig._isPercentageBased;
       var scoreToPass = assessmentConfig._scoreToPass;
+      const correctToPass = assessmentConfig._correctToPass || 0;
 
       var scoreAsPercent = this.get('_scoreAsPercent');
       var score = this.get('_score');
+      const correctAsPercent = this.get('_correctAsPercent');
+      const correctCount = this.get('_correctCount');
 
-      var isPass = isPercentageBased ? (scoreAsPercent >= scoreToPass) : (score >= scoreToPass);
+      const isPass = isPercentageBased ? (scoreAsPercent >= scoreToPass && correctAsPercent >= correctToPass) : (score >= scoreToPass && correctCount >= correctToPass);
```

Comment:
Can we format this ternary statement a bit better and separate it out onto different lines like how you have formatted this one

```
const isPass = !isComplete
? false
: (assessmentsConfig._isPercentageBased !== false)
  ? scoreAsPercent >= scoreToPass && correctAsPercent >= correctToPass
  : score >= scoreToPass && correctCount >= correctToPass;
```